fearless/Common/Helpers/AddressConversion.swift

```diff
@@ -0,0 +1,47 @@
+import Foundation
+import IrohaCrypto
+
+enum ChainConversion {
+    case ethereum
+    case substrate(_ prefix: UInt16)
+}
```

Comment:
I guess renaming it to ChainFormat will make function signatures more self-explanatory